pkg/data/management/role_data.go

```diff
@@ -94,7 +94,7 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("management.cattle.io").resources("kontainerdrivers").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("podsecuritypolicytemplates").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("nodetemplates").verbs("create").
-		addRule().apiGroups("*").resources("secrets").verbs("create").
+		addRule().apiGroups("").resources("secrets").verbs("create").
```

Comment:
This is changing the existing User role's permissions. If we want to allow restricted admin to create standard user, then [this](https://github.com/rancher/rancher/pull/29575/files#diff-1233e0b4cb129841936b478407cce09e86417b3125fd9e68731243adec2d6003R84) permission should be changed to 
```
addRule().apiGroups("*").resources("secrets").verbs("create").
```